src/output/file_name.rs

```diff
@@ -303,59 +303,22 @@ impl<'a, 'dir, C: Colours> FileName<'a, 'dir, C> {
         let file_style = self.style();
         let mut bits = Vec::new();
 
-        self.escape_color_and_hyperlinks(
+        if self.options.embed_hyperlinks == EmbedHyperlinks::On {
+            bits.push(ANSIString::from(format!("\x1B]8;;{}\x1B\x5C", self.file.path.display())));
```

Comment:
`\x1B]8;;{}\x1B\x5C` is used both here and on line 318, maybe we should introduce a `const` to make the hyperlink format "magic string" easier to understand?

Reply:
Perhaps also do a quick check if it appears in other places in the codebase?

Reply:
Fixing this could be it's own separate commit if you'd prefer that.